style.css

```diff
@@ -376,3 +415,145 @@ textarea {
   text-decoration: underline #00d7c3;
   text-decoration-thickness: 2px;
 }
+
+/* FOR DESKTOP VIEW */
+@media only screen and (min-width: 768px) {
+  .union {
+    display: none;
+  }
+
+  .social-arrow {
+    display: none;
+  }
+
+  .headline {
+    background-image: url('images/two-circles-header.png');
+    background-repeat: no-repeat;
+    background-position-y: 100px;
+    background-size: 100%;
+  }
+
+  .work-section {
+    background-image: url('images/Group65@2x.png');
+    background-repeat: no-repeat;
+    background-position-y: 100px;
+    background-size: 100%;
+  }
+
+  .container {
+    padding-left: 230px;
+    padding-right: 230px;
+  }
+
+  .knowledge {
+    display: flex;
```

Comment:
Please ensure to replicate the design as much as possible (e.g., font, colors, images, tests, margins) using [the info from the Figma templates](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

Your Implementation | Figma Design
:---:|:---:
![image](https://user-images.githubusercontent.com/10332499/159935708-ed976060-70b2-4812-9eb9-efdca55dc309.png) | ![image](https://user-images.githubusercontent.com/10332499/159935578-46db80a5-d6d1-4e47-ab83-5fd4733de182.png)

Reply:
@micheaol thank you for your review I am going to change these mentioned changes required but I would like to highlight this issue in instruction where they talked about contact form in desktop version

IMPORTANT NOTE some Figma design templates include two input fields (for first name and last name) in the desktop version and one input field (for full name) in the mobile version (look for those cases in Contact me or Get in touch sections). Please ignore this part of the design template and use only one input for the full name for all versions.